hypershift-operator/main.go

```diff
@@ -207,7 +219,15 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 		EnableOCPClusterMonitoring: opts.EnableOCPClusterMonitoring,
 		CreateOrUpdateProvider:     createOrUpdate,
 		EnableCIDebugOutput:        opts.EnableCIDebugOutput,
-	}).SetupWithManager(mgr); err != nil {
+	}
+	if opts.AWSCredentialsFile != "" {
+		awsSession := awsutil.NewSession("hypershift-operator")
+		awsConfig := awsutil.NewConfig(opts.AWSCredentialsFile, opts.AWSOIDCBucketRegion)
```

Comment:
since we control the environment inside the `hypershift-operator` pod, can we just use the `AWS_SHARED_CREDENTIALS_FILE` env var and let the aws sdk figure it out?

Reply:
Updated to do that